CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## [0.0.1] - TODO: Add release date
```

Comment:
I think can just drop this TODO

Reply:
Formatted it like the one on the main repo